include/ck/utility/get_id.hpp

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include "ck/ck.hpp"
+#include "ck/utility/get_shift.hpp"
```

Comment:
Look like you do not reference get_shift() in the header, could you remove this line?

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/771/commits/b5a9ff28dd5d3b43f82ae68e3b225752665cf52a